docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -169,14 +175,14 @@ it's necessary to follow the naming convention:
 
 .. code-block:: bash
 
-    # filename of the prediction for image FILENAME.EXT
-    .lightly/predictions/${TASK_NAME}/${FILENAME}.json
+    # filename of the prediction for image input/bucket/SUBDIR/FILENAME.EXT
+    .lightly/predictions/${TASK_NAME}/${SUBDIR}/${FILENAME}.json
```

Comment:
I'm not sure about the `${SUBDIR}` notion. It might raise the question if multiple subdirs are also allowed or not (hierarchy). I think with using subdirs in the examples this is already much better. And I'd not add the `${SUBDIR}`.

Reply:
I agree, the `$SUBDIR` is a bit confusing and the other examples should be enough. Thanks for pointing it out!